submarine-workbench/workbench-server/src/main/java/org/apache/submarine/server/GitUtils.java

```diff
@@ -0,0 +1,322 @@
+/*
```

Comment:
I think, Need move `GitUtils.java` to `submarine-workbench/workbench-server/src/main/java/org/apache/submarine/utils` Dir.

Reply:
Done.